src/EPR.Accreditation.API.Common/Dtos/Accreditation.cs

```diff
@@ -32,5 +32,7 @@ public class Accreditation
         public WastePermit WastePermit { get; set; }
 
         public Address LegalAddress { get; set; }
+
+        public decimal? AccreditationFee { get; set; }
```

Comment:
This should just be "Fee". It's already on the Accreditation table, so we know it's AccreditationFee